Automattic/jetpack
# AI Assistant: Files reorganization

Fixes #31337

## Proposed changes:

* Moves and breaks up some files:
```
create-prompt.js              -> lib/prompt/index.ts
get-suggestion-with-stream.js -> lib/suggestions/index.js, lib/suggestions/token.ts
image-with-select.js          -> components/image-with-select/index.js
lib.js                        -> lib/image/index.js
upgrade-prompt.js             -> components/upgrade-prompt/index.js
use-suggestions-from-openai   -> hooks/use-suggestions-from-openai/index.js
ai-control.js                 -> components/ai-control/index.js, components/toolbar-controls/index.js
```

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Add an AI Assistant block
* Check that it works as before, with no console errors
